src/libcollections/vec.rs

```diff
@@ -423,11 +423,18 @@ impl<T> Vec<T> {
         }
     }
 
+    /// Extract a slice containing the entire vector.
+    #[inline]
+    #[unstable(feature = "convert",
+               reason = "waiting on RFC revision")]
+    fn as_slice(&self) -> &[T] {
+        self
+    }
```

Comment:
This shouldn't be necessary, right? If we only intend to call this as a method it should be callable through an inherent method on `[T]`?